### PR TITLE
Fix fragment children

### PR DESCRIPTION
**Pull request by Ayub-Begimkulov**
As was mentioned in #2462 by @joritochip, the fix wasn't working correctly, since we haven't updated all the utils that were working with children to handle fragments.

### CODE REVIEW

**Review by semoal — APPROVED** [review]

**Review by rafaell-lycan — APPROVED** [review]
LGTM

File: src/component/Cell.tsx [hunk 1]
```diff
@@ -5,6 +5,6 @@ import type { FunctionComponent, SVGProps } from 'react';
 
 export type Props = SVGProps<SVGElement>;
 
-export const Cell: FunctionComponent<Props> = (props: Props) => null;
+export const Cell: FunctionComponent<Props> = (_props: Props) => null;
```
Comment (rafaell-lycan) [inline src/component/Cell.tsx:8]: Do the components use `_props` used at all?

Comment (Ayub-Begimkulov) [reply src/component/Cell.tsx:8]: yes, as far as I know
https://recharts.org/en-US/api/Cell

**Comment by Markus-ipse** [comment]
What's the status on this PR? Is there something blocking it or has it just fallen in cracks?

**Comment by Ayub-Begimkulov** [comment]
@Markus-ipse I don't know tbh. It's been a long time since I opened this PR. Seems like this library isn't actively maintained.

**Comment by mohamedmansour** [comment]
@arcthur this pr is good whenever you have time to merge

**Pull request merged**